cli/azd/pkg/project/project_config.go

```diff
@@ -54,3 +55,19 @@ func (p *ProjectConfig) HasService(name string) bool {
 
 	return false
 }
+
+// Retrieves the list of services in the project, in a stable ordering that is deterministic.
+func (p *ProjectConfig) GetServicesStable() []*ServiceConfig {
```

Comment:
We will likely revisit this again shortly, but just wanted to restore the behavior and avoid the regression as described by #1816